src/silx/gui/colors.py

```diff
@@ -534,6 +538,23 @@ def setNormalization(self, norm: str):
             self.__warnBadVmax = True
             self.sigChanged.emit()
 
+    def getSaturation(self) -> int:
+        """Colormap saturation in (0, 100)"""
+        return self._saturation
+
+    def setSaturation(self, saturation: int):
```

Comment:
Maybe add a docstring.
`saturation` name is not explicitly linked to the percentile mode.

What about an explicit name? e.g. `setPercentileAutoscaleSaturation` or `setPercentileAutoscaleParameter` like `setGammaNormalizationParameter`

Reply:
For now I renamed it 'setSaturationAutoscaleParameter'.
Because at this stage this is an int when percentil is expected to be a tuple of float.

Might be renamed again if at the end we go more for 'percentile' (tuple of float) vs 'saturation' (single int)